### PR TITLE
Updated star db

### PR DESCRIPTION
**Pull request by mklewandowski**
This PR adds the new 9K star database to the project and a new magnitude threshold slider to control which stars are visible.  A few notes:
 - We might want to rethink the normalization process in `datacontroller` used to scale stars.  It's based on the min and max magnitudes.  This has changes based on the new star database (range is much larger).  As a result many stars from the previous database now appear larger.  There's nothing wrong with the method, it was simply based on the old database which had a different magnitude range.
 - I left the old star database in as I found it useful as a reference when testing and determining if values are correct in the new database.
- Stars that do not have a parent constellation in the database were given the gameobject and parent container name of "no-const" (didn't want this to be empty string for cases where we are using the scene hierarchy view to examine scene objects).  There is no option in the constellation dropdown to select these no-constellation stars (seemed counter-intuitive based on the function of the dropdown).  They are also given a standard grey color (as per PI request), but moving forward we need to think, in general, about how we color stars (which I believe merits its own story and further discussion of color conventions, color index to RGB conversion, etc.).

### CODE REVIEW

File: Assets/Scripts/Core/Import.cs [hunk 1]
```diff
@@ -7,18 +7,25 @@ public static List<Star> ImportStarData(string sourceData)
...
                 {
-                    string constellation = values[0];
-                    string XByerFlamsteed = values[1];
-                    float RA = float.Parse(values[2]);
-                    float Dec = float.Parse(values[3]);
-                    float Dist = float.Parse(values[4]);
-                    float Mag = float.Parse(values[5]);
-                    Star star = new Star(constellation, XByerFlamsteed, RA, Dec, Dist, Mag);
+                    int Hip = int.Parse(values[0]);
+                    string constellation = values[1];
```
Comment (dstrawberrygirl) [inline Assets/Scripts/Core/Import.cs]: Would be nice to change capitalization to match all other properties that start with a capital letter

Comment (mklewandowski) [reply Assets/Scripts/Core/Import.cs]: Agree.  Fixed.

**Comment by mklewandowski** [comment]
@dstrawberrygirl thanks for the feedback!  I updated all scenes to use the new star DB and tweaked the magnitudeScale so stars don't overlap as much in the stars scene.

As for UI, I think we need to make a general pass at UI updates.  We've been adding quite a bit in, and some of the tools are under-designed, conflict with one another, or haven't had true discussion about how things should work or be implemented.  Nathan liked the magnitude slider so could we run the low/medium/high idea through him before going in that direction?  Same goes with some of the overlapping mouse/touch issues.  I actually don't think the star info box should pop up on mouseover, but should require a more explicit interaction for it to appear (maybe a click brings up the dialog and from the dialog a constellation can be selected).  Maybe we can bring Michael into some talks?  In general, I think some of these larger UI issues should have a group discussion first and then we break them into more manageable stories (rather than trying to attack them here in this PR).  My basic, starter questions would be:
 - what UI tools are needed in each scene?
 - where should these tools appear?
 - how should the user interact with the UI tools and other things in the scene?
 - is there any feedback or suggestions from the team on the current UI?

**Comment by mklewandowski** [comment]
Also tweaked z-order so star info pop-up doesn't cover UI buttons.

**Pull request merged**